blocks/core/ff_module/ff_module-filter/ff_module-filter.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+$(function(){ 
+	$("[data-ff-action='filter-control']").click(function(){
+		var filterDropdown = $(this),
+		      filterIcon = filterDropdown.find("[data-ff-icon='filter-dropdown-icon']"),
+		      filterContent = filterDropdown.closest("[data-ff='filter']").find("[data-ff='filter-content']");
```

Comment:
@roobottom just curious, does it need the `closest("[data-ff='filter']")` bit, is that doing something in a case I can't see just by looking at the code?